cdap-docs/build.sh

```diff
@@ -49,7 +49,9 @@ function usage() {
   echo "    clean     Clean up any previous build's target directories"
   echo "    javadocs  Build Javadocs"
   echo "    licenses  Clean build of License Dependency PDFs"
-  echo "    sdk       Build CDAP SDK"
+  echo "    sdk       Build CDAP SDK (includes the Hydrator plugins if Hydrator plugins source is at"
+  echo "                              '${HYDRATOR_PLUGINS_PATH}'"
```

Comment:
Except this is specifically for Hydrator. By default, if you add the repo locally, then it will use ``hydrator-plugins``. If you want to pass it a different directory, you can, but this is specifically (in this branch) for building Hydrator. The presence of this directory will trigger the function that builds Hydrator.